ocaml/xapi/jbuild

```diff
@@ -71,7 +71,7 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
    pciutil
    pci
    ezxenstore
-   oclock
```

Comment:
Don't you need both `mtime` and `mtime.clock.os` ?

Reply:
I am not sure - the build works as it is now.

Reply:
We do need it I think: https://github.com/dbuenzli/mtime/blob/master/CHANGES.md
I have personally added it to xapi for CBT, so we'll have it anyway :smile: